node_helper.js

```diff
@@ -109,6 +111,11 @@ module.exports = NodeHelper.create({
 	},
 
 	parseInlineJson (str) {
-		return eval?.(`"use strict";(${str})`);
+		try {
+			const object = eval?.(`"use strict";(${str})`);
```

Comment:
why can't we just return this right away?

Reply:
You are right. I implemented this before I implemented the gid check. With the gid check, we don't need to check here again, I think. I have just undone that change.